packages/mongo-nav/src/org-nav/OrgNav.tsx

```diff
@@ -129,18 +129,19 @@ export default function OrgNav({
     >
       <div className={leftSideContainer}>
         <LogoMark height={30} />
-        <MongoSelect
+
+        <OrgSelect
           className={orgSelectContainer}
           data={data}
           current={current}
           constructOrganizationURL={constructOrganizationURL}
-          variant="organization"
           urls={urls}
           onChange={onOrganizationChange}
           isActive={activeNav === 'orgSettings'}
         />
+
         <ul className={ulContainer}>
-          {variant && current.paymentStatus && (
+          {variant && current?.paymentStatus && (
```

Comment:
nit, we should rename variant to `paymentVariant` to disambiguate the multiple variant references we have in this file.

Reply:
👍